src/app/portlets/content-types/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.ts

```diff
@@ -97,6 +97,21 @@ export class ContentTypeFieldsPropertiesFormComponent extends BaseComponent impl
         });
     }
 
+    private formatFormData(): any {
+        const formCopy = Object.assign({}, this.form.value);
```

Comment:
If this object `this.form.value` have a nested object or collection this approach will NOT clone that, the reference to the nested object doesn't change, I think this is the case here, right?